styles.css

```diff
@@ -151,14 +155,12 @@ h4 {
 }
 
 hr {
-  position: static;
   width: 48px;
   height: 4px;
-  left: 139.5px;
-  top: 70px;
   background: #36b37e;
   border-radius: 24px;
-  margin-bottom: 50px;
+  margin-top: 24px;
+  color: #36b37e;
 }
```

Comment:
- [ ] Again Your design matches really well 👏. If you notice the `hr` you have underneath the headings has a default border on it please make a small adjustment in the CSS to remove this border.
In order to try and match the design as close as possible please try to remove even the smallest of discrepancies. This will improve your ability to match a template design using CSS and improve your CSS ability. 
## Your Design (border)
![image](https://user-images.githubusercontent.com/18730106/144668489-d182bd17-b9a0-4889-b7eb-d3c123571cce.png)

## Figma Design (no border)
![image](https://user-images.githubusercontent.com/18730106/144668564-fb1d2d6f-26d5-49b6-b0be-d32fc386add9.png)